src/platformsh.js

```diff
@@ -0,0 +1,54 @@
+/**
+* Read number of CPUs from environment or fallback to the _private_ configuration property
+* Useful for determining the number of processes to fork.
+*/
+const num_of_cpus = () => {
```

Comment:
Errr what's the reason for using `const foo = () => {}` over `function foo () {}`?